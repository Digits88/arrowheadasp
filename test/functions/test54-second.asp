<!-- Test Second function.
-->

Second("12:23:01") = 
	<%= Second("12:23:01") %><br>
Second("12:23:00") = 
	<%= Second("12:23:00") %><br>
Second("12:23:59") = 
	<%= Second("12:23:59") %><br>
IsNull(Second(NULL)) = 
	<%= IsNull(Second(NULL)) %><br>
