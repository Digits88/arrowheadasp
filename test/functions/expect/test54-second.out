<!-- Test Second function.
-->

Second("12:23:01") = 
	1<br>
Second("12:23:00") = 
	0<br>
Second("12:23:59") = 
	59<br>
IsNull(Second(NULL)) = 
	True<br>
